io/post_regional.F90

```diff
@@ -885,11 +896,14 @@ subroutine set_postvars_regional(wrt_int_state,mpicomp,setvar_atmfile,   &
                 line=__LINE__, file=__FILE__)) return  ! bail out
 !           print *,'in post_lam, get land field value,fillvalue=',fillvalue
 
-          ista = lbound(arrayr42d,1)
-          iend = ubound(arrayr42d,1)
-          !$omp parallel do default(none),private(i,j),shared(jsta,jend,ista,iend,spval,arrayr42d,sm,fillValue)
+!JESSE 20220222 keep consistent ista/iend
+!          iista = lbound(arrayr42d,1)
+!          iiend = ubound(arrayr42d,1)
+          iista = ista
+          iiend = iend
```

Comment:
Is `iista` is always equal to`ista`? If it is why not using `ista` in all do loop bounds?

Reply:
Current post only does mpi decomposition in the j direction and most of the variables output the full length in the i direction i=1,im, but a few variables output within i=lbound(arrayr42d,1),ubound(arrayr42d,1) and ista/iend were used.
As we are developing the 2d mpi decomposition, ista/iend are global variables associated with the number of pieces cut in the i direction. I have tested both cases and results look the same. But just keep this commented code so it's easy to track if needed.

Reply:
So, is `iista` always the same as `ista`?